Bottom: 7ac6b55a83c492ac0bb9fb73c8685fc0c5a908a5
Top:    3ede0dc18d0a84ecaf11f31c7c1c52e020881ce0
Author: Jon Smirl <jonsmirl@smirl2.(none)>
Date:   2013-06-29 08:45:57 -0400

Fix 5350 boot


---

diff --git a/target/linux/ramips/base-files/lib/preinit/07_set_preinit_iface_ramips b/target/linux/ramips/base-files/lib/preinit/07_set_preinit_iface_ramips
index db03b6b..0f8da99 100644
--- a/target/linux/ramips/base-files/lib/preinit/07_set_preinit_iface_ramips
+++ b/target/linux/ramips/base-files/lib/preinit/07_set_preinit_iface_ramips
@@ -11,6 +11,10 @@ ramips_set_preinit_iface() {
 		swconfig dev rt305x set reset 1
 	fi
 
+	RT3X5X=`cat /proc/cpuinfo | egrep "(RT3.5|RT5350)"`
+	if [ -n "${RT3X5X}" ]; then
+		swconfig dev rt305x set reset 1
+	fi
 	ifname=eth0
 }
